src/creates/createIssue.ts

```diff
@@ -24,9 +24,9 @@ const createIssueRequest = async (z: ZObject, bundle: Bundle) => {
   const priority = bundle.inputData.priority ? parseInt(bundle.inputData.priority) : 0;
   const estimate = bundle.inputData.estimate ? parseInt(bundle.inputData.estimate) : null;
 
-  const subscriberIds: string[] = []
+  const subscriberIds: string[] = [];
```

Comment:
Is there no working linter in this repo? 🤔

Reply:
Apparently not!